---
permalink: /software/
layout: single
author_profile: true
type: pages
---

## Sequence Manipulation and Viewing  
[Mesquite (GUI)](https://github.com/MesquiteProject/MesquiteCore)  
[Aliview (GUI)](http://www.ormbunkar.se/aliview/)  
[Geneious (GUI)](http://www.geneious.com/)  
[BioPython](http://biopython.org/wiki/Main_Page)  
[DendroPy](http://dendropy.readthedocs.io)  
[PyCogent](http://pycogent.org/)  

## Site Striping and Masking  
[GBlocks](http://molevol.cmima.csic.es/castresana/Gblocks.html)  
[Guidance2](http://guidance.tau.ac.il/ver2/)  
[trimAl](http://trimal.cgenomics.org/)  

## Sequence Alignment  
[Mafft](http://mafft.cbrc.jp/alignment/software/)  
[Prank](http://www.ebi.ac.uk/goldman-srv/prank/prank/)  
[Muscle](http://www.drive5.com/muscle/)  
[hmmer](http://hmmer.janelia.org/)  
[Macse](http://bioweb.supagro.inra.fr/macse/)  
[Blast+](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&DOC_TYPE=Download)  
[LASTZ](http://www.bx.psu.edu/miller_lab/dist/README.lastz-1.02.00/README.lastz-1.02.00a.html#intro)  
[BLAT](https://genome.ucsc.edu/FAQ/FAQblat.html)  

## Model Selection  
[PartitionFinder](https://github.com/brettc/partitionfinder)  
[ProtTest](https://github.com/ddarriba/prottest3)  

## Estimate phylogenies  
[Garli](https://code.google.com/p/garli/)  
[RAxML](https://github.com/stamatak/standard-RAxML)  
[RAxML-ng](https://github.com/amkozlov/raxml-ng)  
[MrBayes](http://mrbayes.sourceforge.net/)  
[ExaML](https://github.com/stamatak/ExaML)  
[ExaBayes](https://www.assembla.com/spaces/exa-bayes/git/source)  
[Astral](https://github.com/smirarab/ASTRAL)  
[BEAST](http://beast2.org/)  
[IQ-TREE](http://www.iqtree.org/#download)  

## Phylogeny and Statistical Visualization (Python modules & R libraries)  
[ETE Toolkit](http://etetoolkit.org)  
[ggtree](http://etetoolkit.org)  
[Matplotlib](http://matplotlib.org)  
[Seaborn](http://seaborn.pydata.org)  
[Bokeh](http://bokeh.pydata.org/en/latest/)  
[ggplot (for Python)](http://yhat.github.io/ggpy/)  
[plotnine](https://plotnine.readthedocs.io/en/stable/)  

## Sequence Simulation  
[INDELible](http://abacus.gene.ucl.ac.uk/software/indelible/)  
[SimPhy](https://github.com/adamallo/SimPhy)  

## NGS Read Visualization and Quality Control  
[FASTX-Toolkit](http://hannonlab.cshl.edu/fastx_toolkit/)  
[FASTQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)  
[Trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)  
[PRINSEQ](http://prinseq.sourceforge.net)  

## NGS Error Correction  
[BLESS](https://sourceforge.net/p/bless-ec/wiki/Home/)  
[Rcorrector](https://github.com/mourisl/Rcorrector)  
[Pollux](https://github.com/emarinier/pollux)  

## NGS Short-Read Aligners  
[Bowtie](http://bowtie-bio.sourceforge.net/index.shtml)  
[BWA](http://bio-bwa.sourceforge.net)  

## NGS Read Alignment Manipulation  
[SAMtools](http://samtools.sourceforge.net)  
[BAMtools](https://github.com/pezmaster31/bamtools)  

## Genome Manipulation Tools  
[BEDtools](http://bedtools.readthedocs.io/en/latest/)  
[gff3](https://pypi.python.org/pypi/gff3/0.2.0)  

## Transcriptome Assembly  
[Trinity](https://github.com/trinityrnaseq/trinityrnaseq/wiki)  
[Velvet](https://github.com/dzerbino/velvet)  
[SOAPdenovo-Trans](http://soap.genomics.org.cn/SOAPdenovo-Trans.html)  

## Genome Assembly  
[SOAPdenovo2](https://github.com/aquaskyline/SOAPdenovo2)  
[ALLPATHS-LG](http://software.broadinstitute.org/allpaths-lg/blog/?page_id=12)  
[DISCOVAR de novo](https://software.broadinstitute.org/software/discovar/blog/?page_id=98)  
[SPAdes](http://bioinf.spbau.ru/spades)  

## Geo Mapping  
[Basemap Maplotlib Toolkit](http://matplotlib.org/basemap/)  
[Bokeh](http://bokeh.pydata.org/en/latest/)  
[Vincent](https://vincent.readthedocs.io/en/latest/)  

## Python IDE  
[RODEO](https://www.yhat.com/products/rodeo)

## Sanger primers and hybrid capture probes
[Primer3.py](https://libnano.github.io/primer3-py/index.html#)
[BaitFisher](https://github.com/cmayer/BaitFisher-package)
